Doc/library/datetime.rst

```diff
@@ -570,23 +569,23 @@ Instance attributes (read-only):
 
 .. attribute:: date.month
 
-   Between 1 and 12 inclusive.
+   Between ``1`` and ``12`` inclusive.
```

Comment:
This is not a code. They are just numbers.

Reply:
The change has been reverted; marking as resolved.